lib/cyclonedx/report_rust_crates_cyclonedx.rb

```diff
@@ -1,16 +1,10 @@
 module Cyclonedx
   class ReportRustCrates < Base
+    TYPE = "cargo".freeze
+    UNLOCKED_DEPENDENCY_FILE = "".freeze
```

Comment:
The default value of `UNLOCKED_DEPENDENCY_FILE` is `N/A` so if there's no actual unlocked_dependency_file you can just use that default value

Reply:
will update!